src/app-layer-modbus.c

```diff
@@ -274,6 +274,24 @@ static LoggerId ModbusGetTxLogged(void *alstate, void *vtx)
     return tx->logged;
 }
 
+static void ModbusSetTxDetectFlags(void *vtx, uint8_t dir, uint64_t flags) {
```

Comment:
style: { on a newline

Reply:
I see others like this (like in enip). Will review everywhere for this.

Ironically I believe most of that was a complete auto-suggestion by TabNine heh.